tmexp/merge.py

```diff
@@ -30,11 +30,7 @@ def merge_datasets(
     for input_path in input_paths:
         with open(input_path, "rb") as fin:
             input_dict = pickle.load(fin)
```

Comment:
could you add a warning, or possibly raise an error here:
```
for repo, refs in input_dict["ref"].items():
    if len(refs) > 1:
        logger.warning("Found %d tagged references for repository %s !")
```
Feel free to put something more explicit, but as we should never merge repos if they have multiples refs we should not allow it imo, or at least warn the user

Reply:
As discussed on Slack, we will allow multiple refs when merging datasets to accommodate our future main use-case but we'll keep the warning to notify the user that several revisions are being merged.